shell/demo/demo5.ps1

```diff
@@ -0,0 +1,173 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true, HelpMessage="Prelude Token")]
+    [String]$preludeToken
+)
+
+function FromEnv { param ([string]$envVar, [string]$default)
+    $envVal = [Environment]::GetEnvironmentVariable($envVar, "User")
+    if ($envVal) { return $envVal } else { return $default }
+}
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="ca9b22be-93d5-4902-95f4-4bc43a817b73"
+$TEST_INTRO="Will your computer quarantine a malicious python file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious python file was used by the malware authors Portugal and Brazil
+during a supply chain attack in February 2023.
+
+This test will attempt to download a defanged malicious file to your disk - containing a signature for the Colour-Blind malware used in a recent supply chain attack - to see how your endpoint defenses will respond. 
```

Comment:
This test will attempt to download a defanged malicious `python` file

Reply:
resolved